src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Form } from './components/Form/Form';
+import { getUserById } from './services/user';
+import { EnrichedTodo } from './types/EnrichedTodo';
+import { useState } from 'react';
+
+export const userTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodoId(todo: EnrichedTodo[]) {
+  const maxId = Math.max(...todo.map(singeTodo => singeTodo.id));
```

Comment:
The `getNewTodoId` function should handle cases where the `todos` array might be empty. Use a default value in `Math.max` to prevent it from returning `-Infinity`, which would cause incorrect ID generation. For example, `Math.max(0, ...todo.map(singeTodo => singeTodo.id))`.